yurasel228/hw 7/hw 3.py

```diff
@@ -0,0 +1,21 @@
+def string(string:str) -> None:
```

Comment:
look at this implementation:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result